docker/maistra-builder_1.1.Dockerfile

```diff
@@ -1,19 +1,25 @@
-FROM registry.access.redhat.com/ubi8:8.0
+FROM centos:8.1.1911
```

Comment:
Just `centos:8` isn't better? So, for every build it will get latest centos 8 minor release for free.

Also, I think you could put something like a `dnf update` as well, just to make sure the image contains the latest patches. I think that will also fix those warnings in Quay.

Reply:
Good point, I changed it to `centos:8` and added a `dnf update` at the end. It fixes all warnings except for 1 which apparently hasn't made it into centos yet